gammapy/maps/hpx.py

```diff
@@ -1731,6 +1731,23 @@ def solid_angle(self):
         import healpy as hp
         return Quantity(hp.nside2pixarea(self.nside), 'sr')
 
+    def __repr__(self):
+        """String representation of the MapAxis object"""
```

Comment:
Suggest to remove all the docstrings on the `__repr__` methods.
It's just something Python coders have to know, and users won't see them anyways.
I'd prefer not to add them 100x in Gammapy.
Also: you currently have copy & paste errors, you always say "MapAxis".